### PR TITLE
Quick copy updates to the README file

### PR DESCRIPTION
**Pull request by danilo-leal**
Just simplifying a fine-tuning it a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
+Follow the steps:
 
-## Submission
+- Clone the repo: `git clone git@github.com:mui/tech-challenge-design-engineer.git`
+- Install the dependencies: `yarn`
+- Start Next.js: `yarn dev`
+- Open http://0.0.0.0:3002/
 
-Instructions:
+## Submission instructions
```
Comment (oliviertassinari) [inline README.md:61]: There is a potential to apply the same improvements in

- https://github.com/mui/tech-challenge-full-stack#submission
- https://github.com/mui/tech-challenge-react#submission

File: README.md [hunk 2]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
+Given that the Design Engineer position sits right at the intersection between engineering and design, this challenge is tailored to demonstrate your skills in both areas.
 Our challenge is to **develop a stacked Snackbar** component.
 
 Here's a reference design:
 
 <img alt="Snackbars example design" src="./public/snackbar-reference.gif" width="465" height="392" />
 
 Note that it's only an example.
-We encourage you to add your creative spin to the design, we want to see your design taste!
+We encourage you to add your creative spin to the design; we want to see your design taste!
```
Comment (mapache-salvaje) [inline README.md]: punctuation nitpick 😜

```suggestion
We encourage you to add your creative spin to the design—we want to see your design taste!
```

File: README.md [hunk 3]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
 
-## Objective
+## Requirements
 
-We do have a few requirements, though:
-
-- **Stack:** You can use Base UI's [`UnstyledSnackbar`](https://mui.com/base/react-snackbar/) component or another headless component library as the foundation for it.
+- **Stack:** Use Base UI's [`UnstyledSnackbar`](https://mui.com/base/react-snackbar/) component or another headless component library as its foundation.
```
Comment (mapache-salvaje) [inline README.md]:
```suggestion
- **Stack:** Use the Base UI [Snackbar](https://mui.com/base/react-snackbar/) or a similar headless component from a library of your choice as the foundation.
```

File: README.md [hunk 4]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
 - **Interactivity:**
   - The other Snackbar instances should be visible after some interaction, either clicking a button or hovering the stack.
   - They should be kept open if the focus is inside them. Conversely, they should collapse as the focus moves out of it.
```
Comment (mapache-salvaje) [inline README.md]:
```suggestion
  - They should stay open when focused, and collapse when the focus moves away.
```

File: README.md [hunk 5]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
-  - There should be the possibility to close them by clicking on a close button.
-  - _Bonus:_ adding an option for it to be closed after some configurable time interval. It should still be compatible with [WCAG 2.2.1](https://www.w3.org/TR/WCAG21/#enough-time).
+  - Closing them should be possible by clicking on a close button.
```
Comment (mapache-salvaje) [inline README.md]:
```suggestion
  - Each instance should have a functional close button.
```

File: README.md [hunk 6]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
+  - _Bonus:_ adding an option to close it after some configurable time interval. It should still be compatible with [WCAG 2.2.1](https://www.w3.org/TR/WCAG21/#enough-time).
```
Comment (mapache-salvaje) [inline README.md]:
```suggestion
  - _Bonus:_ Add an option to make the Snackbars close automatically after some configurable time interval. This should be compatible with [WCAG 2.2.1](https://www.w3.org/TR/WCAG21/#enough-time).
```

File: README.md [hunk 7]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
 - **Transitions:** the Snackbars should appear and disappear gracefully on the screen, using an animation.
 - **Dark mode:** make sure to support light and dark modes.
```
Comment (mapache-salvaje) [inline README.md:44]: These are not "bonuses" right? If so, maybe they should go above the one Bonus on the list for the sake of clarity?

File: README.md [hunk 8]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
 ## Work environment
 
-To save you time, a working environment was created with Next.js/TypeScript/eslint, etc.
-You can install this environment by following these steps:
-
-- clone the repo: `git clone git@github.com:mui/tech-challenge-design-engineer.git`
-- install the dependencies: `yarn`
-- start Next.js: `yarn dev`
-- open http://0.0.0.0:3002/
+To save you time, we created a working environment with Next.js, Typescript, eslint, and others, that you should install.
```
Comment (mapache-salvaje) [inline README.md]:
```suggestion
To save you time, we created a working environment with Next.js, Typescript, eslint, and other complementary dependencies.
```

File: README.md [hunk 9]
```diff
@@ -19,49 +19,45 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
 
-## Objective
+## Requirements
 
-We do have a few requirements, though:
-
-- **Stack:** You can use Base UI's [`UnstyledSnackbar`](https://mui.com/base/react-snackbar/) component or another headless component library as the foundation for it.
+- **Stack:** Use Base UI's [`UnstyledSnackbar`](https://mui.com/base/react-snackbar/) component or another headless component library as its foundation.
 - **Interactivity:**
   - The other Snackbar instances should be visible after some interaction, either clicking a button or hovering the stack.
```
Comment (mapache-salvaje) [inline README.md]:
```suggestion
  - The other Snackbar instances should be visible after some interaction, either clicking a button or hovering over the stack.
```

**Review by mapache-salvaje — APPROVED** [review]
looks great!

**Pull request merged**